concepts/booleans/.meta/config.json

```diff
@@ -1,8 +1,7 @@
 {
-    "blurb": "As in most other programming languages, a boolean type in Cairo has two possible values: `true` and `false`.",
+    "blurb": "As in most other programming languages, a Boolean type in Cairo has two possible values: true and false.",
```

Comment:
Remove all changes to:
- booleans
- felts
- integers

We'll address some of these in a later PR.